### PR TITLE
W-Y platoon

### PR DESCRIPTION
**Pull request by bearrrrrrrr**
# About the pull request
TAKEN OVER BY ANDRO THE BETEL!

WY Platoon AZURE-15

new map - USCSS Apostasy 
new guns - m39 duplex, M240 firing globs
new platoon - pmc's

# Explain why it's good for the game
wy and wy pmcs are quite prominent in the aliens lore and could offer gms and players more opportunities for cool stories and roleplay

### CODE REVIEW

**Comment by morrowwolf** [comment]
Hit "ready for review" when you're ready for review

File: code/game/jobs/job/marine/squads.dm [hunk 1]
```diff
@@ -296,6 +296,17 @@
 	name = "Taskforce White"
 	locked = TRUE
 	faction = FACTION_WY_DEATHSQUAD
+
+
+//##############################
+
+/datum/squad/pmc/pve
```
Comment (AndroBetel) [inline code/game/jobs/job/marine/squads.dm:303]: should be a subtype of /datum/squad/marine, see my UPP platoon for example

**Review by AndroBetel — CHANGES_REQUESTED** [review]

**Comment by bearrrrrrrr** [comment]
??????????

**Comment by KoishiVibe** [comment]
are ya winnin fren?

File: code/modules/gear_presets/pmc_pve.dm [hunk 2]
```diff
@@ -12,6 +12,126 @@
 	rank = JOB_SQUAD_MARINE
 	flags = EQUIPMENT_PRESET_START_OF_ROUND|EQUIPMENT_PRESET_MARINE
 	skills = /datum/skills/pfc
+	uses_special_name = TRUE
+
+/datum/equipment_preset/uscm/pmc/load_name(mob/living/carbon/human/new_human, randomise)
```
Comment (AndroBetel) [inline code/modules/gear_presets/pmc_pve.dm:17]: this is fucked up. but it's cool. but it's still fucked up.

If anything - i dont mind removing it and using it as OOC guide for PMC names

**Review by AndroBetel — APPROVED** [review]
*approving myself, woah*

**Review by DexterDude — CHANGES_REQUESTED** [review]
Map review time!!!

This is just an initial review, going to do a deeper check in a few when I have more free time.

- I think the singularity part of the map being present is weird. Generally shipmaps try to imply that the ship is much larger than the map actually is. Singularity being present in a hangar bay is a bit odd.
-  should have vendors for coffee/cigarettes for rp
- hangar is too cramped, it needs a bit more space to allow for a large vehicle to get around the hangar bay and enter through the back without having to drive over the landing pad, just in case GMs decide to use a larger dropship.
- Map has no real cargo area for incoming equipment and also has no launch pad & computer

reviewspot1

reviewspot2

reviewspot3

reviewspot4

reviewspot5

reviewspot6

if you disagree with any of this (like if you believe certain things are intentional) do let me know

**Comment by DexterDude** [comment]
forgot to mention, but this podlock south-east in the hangar is also un-named, should be named hangar lockdown or something similar

hangardoors

**Comment by AndroBetel** [comment]
map review changes done

**Comment by DexterDude** [comment]
Still need to do an in depth review, but you did miss the area around the W-Y logo in the canteen. It needs the rounded corners.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by Doubleumc — CHANGES_REQUESTED** [review]
Change dropship name to something other than "Orphan Stomper". Too goofy. Suggestions: "Problem Solver", "Guns For Hire", "Peacemaker", "Breadwinner", "Cash Flow" -- anything you could put on a financial report and not get yelled at by the director.

Other than that, looks good to me.

**Comment by KoishiVibe** [comment]
> Change dropship name to something other than "Orphan Stomper". Too goofy. Suggestions: "Problem Solver", "Guns For Hire", "Peacemaker", "Breadwinner", "Cash Flow" -- anything you could put on a financial report and not get yelled at by the director.
> 
> Other than that, looks good to me.

Probably could also toss some edge in there. Reaper's Stockbroker, whatever.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by Doubleumc** [comment]
Feel free to reopen when the issues have been addressed.

**Pull request closed without merging**